templates/engage/industrial-drones-case-study.html

```diff
@@ -1,8 +1,4 @@
-{% extends "engage/base_engage.html" %}
-
-{% block meta_description %}This case study presents how Apellix, an industrial drone company, engineers safer work environments with Ubuntu.{% endblock %}
-
-{% block title %}Apellix engineers safer work environments with Ubuntu powered aerial robotics{% endblock %}
+{% extends_with_args "engage/base_engage.html" with title="Apellix engineers safer work environments with Ubuntu powered aerial robotics" meta_image="5fd2e1aa-apellix_im3.png" meta_description="This case study presents how Apellix, an industrial drone company, engineers safer work environments with Ubuntu." %}
```

Comment:
You need to use the full path for the image: "https://assets.ubuntu.com/v1/5fd2e1aa-apellix_im3.png"

Reply:
@anthonydillon I think I fixed that here - https://github.com/canonical-web-and-design/ubuntu.com/blob/master/templates/templates/_extra_metadata.html, it adds the path if it is missing